fasm/__init__.py

```diff
@@ -10,12 +10,18 @@
 # SPDX-License-Identifier: ISC
 
 from __future__ import print_function
```

Comment:
Could you rework this so there is an option to use either the older `textx` parser or the new antlr version?

Reply:
> Should delete textx schema?

I thought it might be useful to keep it for reference. Maybe not.

Reply:
I've removed the text.tx schema for now, and created an issue: https://github.com/SymbiFlow/fasm/issues/29